source/server/server.cc

```diff
@@ -58,7 +58,10 @@ InstanceImpl::InstanceImpl(
     : init_manager_(init_manager), workers_started_(false), live_(false), shutdown_(false),
       options_(options), validation_context_(options_.allowUnknownStaticFields(),
                                              !options.rejectUnknownDynamicFields()),
-      time_source_(time_system), restarter_(restarter), start_time_(time(nullptr)),
+      time_source_(time_system), restarter_(restarter),
+      start_time_(std::chrono::duration_cast<std::chrono::seconds>(
```

Comment:
Can you factor some of this out with a helper similar to DateUtil::nowToMilliseconds, which is in source/common/common/utility.h ?

Thanks!